src/animate/load.ts

```diff
@@ -167,11 +178,19 @@ export function load(scene: AnimateAsset, optionsOrComplete?: Complete | LoadOpt
                     data = metadata.default;
                 }
             }
-            promises.push(Assets.load({ alias: [id], src: basePath + assets[id], data }).then((loadedAsset) =>
+            promises.push(Assets.load({ alias: [id], src: basePath + assets[id], data }, progress =>
             {
+                if (optionsOrComplete && typeof optionsOrComplete !== 'function')
+                {
+                    if (optionsOrComplete.progress) optionsOrComplete.progress(progress * loadedAssets / totalAssets);
+                }
+            }).then((loadedAsset) =>
+            {
+                loadedAssets++;
+
                 if (!loadedAsset)
                 {
-                    return; // not sure if this can evet happen
+                    return; // not sure if this can event happen
```

Comment:
While we are fixing typos, the typo here was that it should be `even`.

Reply:
Done